.eslintrc.js

```diff
@@ -74,6 +74,7 @@ module.exports = {
                 '@typescript-eslint/no-unsafe-member-access': 'off',
                 '@typescript-eslint/no-unsafe-assignment': 'off',
 
+                '@typescript-eslint/array-type': ['error', {default: 'array-simple'}],
```

Comment:
Thought we decided to allow both usages?

Reply:
Message from @fabioh8010:

> TS arrays -> https://github.com/Expensify/App/pull/21050#discussion_r1235090963
Hayata suggested to enable this rule -> https://typescript-eslint.io/rules/array-type/#array-simple
I did some tests and look really good, wdyt?

I thought you suggested this change @hayata-suenaga 😅. Me and Fabio agreed that it is great:
![image](https://github.com/Expensify/App/assets/46095609/d190b614-cbbe-4904-b8b2-baf000be3080)

Reply:
aaahhhh I completely misunderstood! never mind now I remember 😅